xAPI.md

```diff
@@ -302,7 +302,44 @@ As a rule of thumb, if the guideline appears technical or seems to be a requirem
 as such. This is especially true of longer, more, detailed explanations and of tables, each of which would 
 be unintuitive and/or lengthy to dissect into a list of requirements.
 
-## 3.0 Guidenelines for Implementing Technical Specificationss Like xAPI
+## 3.0 Guidelines for Implementing Technical Specifications Like xAPI
```

Comment:
Maybe merge with 2.0 , all under "How to Read This Specification"  2.1 = How to interpret language 2.2 - Basic rules and guidelines (fuzzy titles for now)